libraries/networking/src/udt/SendQueue.cpp

```diff
@@ -63,6 +64,46 @@ void SendQueue::queuePacket(std::unique_ptr<Packet> packet) {
     }
 }
 
+void SendQueue::queuePacketList(std::unique_ptr<PacketList> packetList) {
+    Q_ASSERT(packetList->_packets.size() > 0);
+
+    {
+        auto messageNumber = getNextMessageNumber();
+
+        if (packetList->_packets.size() == 1) {
+            auto packet = packetList->_packets.front().get();
```

Comment:
It's a unique_ptr so I would either need to do that or use a reference (`auto& packet = packetList->_packets.front()`) - I'm okay with that too.